docs/admin/memory.rst

```diff
@@ -32,7 +32,7 @@ Translation memory scopes
    In earlier versions translation memory could be only loaded from a file
    corresponding to the current imported translation memory scope.
 
-The translation memory scopes allow both privacy and sharing translations.
+The translation memory scopes ensure privacy, while enabling sharing translations.
```

Comment:
```suggestion
The translation memory scopes allow for privacy when sharing translations.
```
Privacy is never ensured, as that is a dangerous claim.
Having any privacy isn't a product of turning on the translation memory sharing.
This makes sense to me:
`Sharing the translation memory has fine-tuned settings to adjust the scope of how and where sharing occurs.`

Reply:
You are connecting here two things into one. There is a choice to keep TM private or share it. Together it won’t go well.

Reply:
made the change to:

```
The translation-memory scopes ensure privacy for different projects and users. 
Sharing of translations is also available.
```